getambassadorio_blc.py

```diff
@@ -54,17 +54,23 @@ def is_internal_domain(self, netloc: str) -> bool:
 
     def product_should_skip_link(self, link: Link) -> bool:
         links_to_skip = [
-            'http://verylargejavaservice:8080/',
-            'https://blog.getambassador.io/search?q=canary',
-            'https://app.datadoghq.com/apm/traces/',
-            'http://web-app.emojivoto/',
-            'http://web-app.emojivoto/leaderboard/',
-            'http://verylargejavaservice.default:8080/color',
+            'http://localhost:3000/',
+            'http://localhost:3000/color',
             'http://localhost:8080/',
             'http://localhost:8083/',
             'http://localhost:8083/leaderboard/',
             'http://verylargejavaservice.default:8080/',
+            'http://verylargejavaservice.default:8080/color',
+            'http://verylargejavaservice:8080/',
+            'http://web-app.emojivoto/',
+            'http://web-app.emojivoto/leaderboard/',
+            'https://app.datadoghq.com/apm/traces/',
+            'https://blog.getambassador.io/search?q=canary',
+            'https://kubernetes.default',
+            'https://martinfowler.com/articles/microservice-testing/#testing-progress-1',
+            'https://tanzu.vmware.com/kubernetes-grid',
```

Comment:
is this site responding a 403? if this is the case we need to validate that response because we don't know if in the future we are getting a 404 but we skipping it

Reply:
for ‘https://martinfowler.com/articles/microservice-testing/#testing-progress-1' the blc didn’t find the anchor, because It doesn’t exist.
for ‘https://tanzu.vmware.com/kubernetes-grid’ I received a 200 status, but Its marked as 404 in blc2

Reply:
then these are no exception, we need to handle them in a better way, for `‘https://martinfowler.com/articles/microservice-testing/#testing-progress-1` is a broken link because it doesn't exist, and for `https://tanzu.vmware.com/kubernetes-grid` we need to explore why we're getting a 404 if the link works

Reply:
I think that we need to add an exception for the link https://martinfowler.com/articles/microservice-testing/#testing-progress-1 because its an external link and works but It doesn’t have the anchor inside the page, that’s why we got '(fragment: no element with that id/name=''testing-progress-1'')'